python/ola/PidStore.py

```diff
@@ -928,17 +973,32 @@ def __init__(self):
     self._manufacturer_id_to_name = {}
 
   def Load(self, pid_files, validate=True):
-    """Load a PidStore from a file.
+    """Load a PidStore from a list of files.
 
     Args:
-      pid_files: A list of PID files on disk to load
+      pid_files: A list of PID files on disk to load.  If
+      a file "overrides.proto" is in the list it will be loaded
+      last and its PIDs override any previously loaded.
       validate: When True, enable strict checking.
     """
     self._pid_store.Clear()
+
+    raw_list = pid_files
+    pid_files = []
+    override_file = None
+
+    for f in raw_list:
+      if os.path.basename(f) == "overrides.proto":
+        override_file = f
+      else:
+        pid_files.append(f)
```

Comment:
I think you may want to skip manufacturer_names.proto here too, as per the C++:
https://github.com/OpenLightingProject/ola/blob/e90be831d23aad074cc7efd7ebc75f9d6daf7b75/common/rdm/PidStoreLoader.cpp#L93-L95

Reply:
Interesting.  That's in main but not 0.10.  Put it in and will see if anything changes...

Reply:
Yeah 0.10 doesn't have the concept of manufacturer names.

Reply:
Nothing should change, but if you grabbed the manufacturer_names.proto file I think it would fail to load as it would see duplicates.